src/main/java/dataStructures/trie/Trie.java

```diff
@@ -15,99 +15,177 @@ public Trie() {
         root = new TrieNode();
     }
 
+    private class TrieNode {
+        // CHECKSTYLE:OFF: VisibilityModifier
+        public Map<Character, TrieNode> children; // or array of size 26 (assume not case-sensitive) to denote each char
+        // CHECKSTYLE:OFF: VisibilityModifier
+        public boolean isEnd; // a marker to indicate whether the path from the root to this node forms a known word
```

Comment:
perhaps it would be clearer to indicate that the fields are set to public for decreased verbosity here, given that typical best practices would still have private fields with getter/setter methods

Reply:
okay, resolved